plugins/active_directory_ldap/komand_active_directory_ldap/actions/query/action.py

```diff
@@ -20,17 +20,15 @@ def run(self, params={}):
         query = query.replace("\\>=", ">=")
         query = query.replace("\\<=", "<=")
 
-        # find pars of `(` `)`
-        pairs = formatter.find_parentheses_pairs(query)
-
         # replace ( and ) when they are part of a name rather than a search parameter
-        escaped_query = formatter.escape_brackets_for_query(query, pairs)
-        self.logger.info(f"Escaped query: {escaped_query}")
+        escaped_query = formatter.escape_brackets_for_query(query)
+        self.logger.info("Escaped query: %s", escaped_query)
```

Comment:
```suggestion
        self.logger.info(f"Escaped query: {escaped_query}")
```
If you have changed this because of complaints on the logger about f strings, feel free to ignore these.

Reply:
Yea. Exactly. It seems that Static Code Analysis check is failing if I use f-string here 😞